webpack.config.js

```diff
@@ -0,0 +1,233 @@
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const PeerDepsExternalsPlugin = require('peer-deps-externals-webpack-plugin');
+const postcssImport = require('postcss-import');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssReporter = require('postcss-reporter');
+const postcssCustomProperties = require('postcss-custom-properties');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const postcssPostcssColorModFunction = require('postcss-color-mod-function');
+
+const browserslist = {
+  development: ['chrome', 'firefox'].map(
+    browser => `last 2 ${browser} versions`
+  ),
+  production: ['>1%', 'not op_mini all', 'ie 11'],
+};
+
+const sourceFolders = [
+  path.resolve(__dirname),
+  path.resolve(__dirname, '../examples'),
+  path.resolve(__dirname, '../src'),
+];
+
+module.exports = {
+  entry: {
+    main: './src/index.js',
+    utils: './src/utils/index.js',
+  },
+  mode: 'production',
+  output: {
+    path: path.join(__dirname, 'dist'),
+    library: 'McUiKit',
+    libraryTarget: 'umd',
+    filename: chunkData =>
+      chunkData.chunk.name === 'main' ? 'index.js' : '[name]/index.js',
+  },
+  devtool: 'source-map',
+  plugins: [
+    new CleanWebpackPlugin(['dist']),
+    // We copy everything we want to be in the bundle to "dist",
+    // as we publish from "dist".
+    //
+    // We do this to enable consumers to import without having "lib"
+    // in the path, and to avoid having to bundle onto the main folder.
+    //
+    // This is also why the "main" field in package.json says
+    // "index.js" instead of "dist/index.js", as package.json gets copied
+    // to "dist" as well.
+    new CopyWebpackPlugin([
+      { from: './package.json' },
+      { from: './README.md' },
+    ]),
+    // Adds peerDependencies as externals
+    // See: https://webpack.js.org/guides/author-libraries/#externalize-lodash
+    new PeerDepsExternalsPlugin(),
+  ],
+  module: {
```

Comment:
The `module` part was copied from `.storybook/webpack.config.js`. I only removed the `.md` loaders as we don't need them here. Further optimizations may be possible.